Resources/Prototypes/game_presets.yml

```diff
@@ -39,6 +39,24 @@
   rules:
     - Secret
 
```

Comment:
It would be useful to document in comments why this exists, i.e.
```suggestion
# For admins to set extended to run events without spoiling any of the players
```
So that folks who wander here don't get confused, or we all forget.